packages/mobx/src/api/observable.ts

```diff
@@ -175,17 +180,15 @@ export interface IObservableFactory extends Annotation, PropertyDecorator {
 }
 
 const observableFactories: IObservableFactory = {
-    box<T = any>(value?: T, options?: CreateObservableOptions): IObservableValue<T> {
+    box<T = any>(value: T, options?: CreateObservableOptions): IObservableValue<T> {
         const o = asCreateObservableOptions(options)
         return new ObservableValue(value, getEnhancerFromOptions(o), o.name, true, o.equals)
     },
     array<T = any>(initialValues?: T[], options?: CreateObservableOptions): IObservableArray<T> {
         const o = asCreateObservableOptions(options)
-        return (
-            globalState.useProxies === false || o.proxy === false
-                ? createLegacyArray
-                : createObservableArray
-        )(initialValues, getEnhancerFromOptions(o), o.name)
+        return (globalState.useProxies === false || o.proxy === false
```

Comment:
Was it added by prettier?

Reply:
Will revert all stylistic changes, they were made but some auto formatter.